enumeratum/src/main/scala/com/monovore/decline/enumeratum/ValueEnumArgument.scala

```diff
@@ -0,0 +1,28 @@
+package com.monovore.decline
+package enumeratum
+
+import cats.data.{Validated, ValidatedNel}
+
+import _root_.enumeratum.values._
+
+import scala.reflect.ClassTag
+
+private[enumeratum] final class ValueEnumArgument[A, Entry <: ValueEnumEntry[A]](
+    enum: ValueEnum[A, Entry],
+    baseArgument: Argument[A],
+    ct: ClassTag[Entry]
+  ) extends Argument[Entry] {
+  
+  override def defaultMetavar: String = ct.runtimeClass.getSimpleName().toLowerCase()
```

Comment:
I'd like to avoid a dependency on the runtime class name... could we use the `baseArgument.defaultMetavar` here and maybe just something generic like `value` below?